packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -202,7 +213,8 @@ export class CDKDeployer implements BackendDeployer {
       await childProcess;
       return cdkOutput;
     } catch (error) {
-      // swallow execa error which is not really helpful, rather throw stderr
+      // swallow execa error which is not really helpful rather throw the entire stderr
+      // for clients to figure out what to do with it.
       throw new Error(aggregatedStderr);
```

Comment:
it might be good to keep `error` as `cause` or some other field in error and perhaps print it with `--verbose` when we get there.

Reply:
might be worth adding one more line to the comment that explains that.